src/components/Profile/views/RegionsVerification.view.tsx

```
@@ -0,0 +1,80 @@
+                    }
+                    position={getCardPosition(index, regions.length)}
+                    title={region.name}
+                    onClick={() => {
+                        if (isLocked) {
+                            router.push(`/profile/identity-verification/${region.path}`)
+                        }
+                    }}
+                    description={region.description}
+                    descriptionClassName="text-xs"
+                    rightContent={!isLocked ? <Icon name="check" className="size-4 text-success-1" /> : null}
+                />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)